app/views/user_mailer/invited_to_collection_notification.html.erb

```diff
@@ -1,15 +1,12 @@
 <% content_for :message do %>
-  <p>Dear <%= style_bold(@user.default_pseud.byline) %>,</p>
+  <p><%= t(".greeting", name: style_bold(@user.default_pseud.byline)).html_safe %></p>
 
-  <p>The collection maintainers of <%= style_link(@collection.title, collection_url(@collection)) %> would
-    like to include your work (<%= style_link(@work.title.html_safe, work_url(@work)) %>) in their collection!</p>
+  <p><%= t(".part1", collection: style_link(@collection.title, collection_url(@collection)).html_safe, work: style_link(@work.title.html_safe, work_url(@work)).html_safe).html_safe %></p>
 
-  <p>To approve or reject this request, please visit your <%= style_link("Collection Items page", user_collection_items_url(@user)) %>.</p>
+  <p><%= t(".html.part2", link: style_link(t(".html.link_text").html_safe, user_collection_items_url(@user)).html_safe).html_safe %></p>
 
-  <p>If in future you would prefer to automatically approve requests to add your
-    works to collections, please edit your <%= style_link("Preferences", "#{root_url}users/#{@user.default_pseud.name}/preferences") %>
-    and tick the box next to: <%= style_quote("\"Automatically agree to your work being collected by others in the Archive.\"") %>
+  <p><%= t(".html.part3", prefs: style_link(t(".html.preferences"), user_preferences_url(@user)), quote: style_quote(t(".html.quote_link")).html_safe).html_safe %>
```

Comment:
`quote_link` might not be the best name, since it's not a link. Just something like `preference_name` or `auto_agree_preference` is probably fine, but `quoted_preference_name` works too.

Reply:
Ah yes, I spent a long time trying to think up a good name for that, I like `quoted_preference_name`, thanks.